artifactory.py

```diff
@@ -1657,17 +1714,26 @@ def find_permission_target(self, name):
             return obj
         return None
 
-    def writeto(self, out, chunk_size=None):
+    def writeto(
+        self,
+        output,
+        chunk_size=1024,
+        progress_func=print_download_progress,
+        *args,
+        **kwargs
+    ):
         """
-        Writes artifact to file descriptor in chunks
+        Downloads large file in chunks and prints progress. Suppress print by setting progress_func=none
```

Comment:
>prints progress

It's true by default, but not always. What about `and call a progress function`? It's true.

>Suppress print by setting progress_func=none
Lets move it to `:param progress_func` description and here we can mention that by default it's print and if you want to - provide your function or None to suppress logging (it shouldn't be print, right?)